WcaOnRails/config/locales/fi.yml

```diff
@@ -155,14 +155,14 @@ fi:
       other: '%{count} yritystä < %{time}'
     #original_hash: b49d930
     moves:
-      other: '%{count} yritystä < %{moves} siirtoa'
-      one: '%{count} yritys < %{moves} siirtoa'
       zero: '%{count} yritystä < %{moves} siirtoa'
+      one: '%{count} yritys < %{moves} siirtoa'
```

Comment:
@jonatanklosko, it looks like nothing changed here, but the order of the keys did change. Is it possible that https://github.com/jonatanklosko/internationalize outputs these pluralization (key, value) pairs in an inconsistent fashion?